### ISSUE TITLE
Video upload Issue - wandb

### ISSUE THREAD

**Issue by tbasaklar** [vwxyzjn/cleanrl#397]
Hi,

I was following the "Experiment Tracking" at https://docs.cleanrl.dev/get-started/experiment-tracking/

In this example, the captured videos are also uploaded to wandb. However, although I followed the exact same command, my wandb project missed the "Media" folder in "Files", hence the videos do not show up in the browser. What could be the issue here? Could you try to rerun the command in the docs page of cleanRL and see if the videos are uploaded on your side?

Thanks,

**Comment by vwxyzjn** [comment]
You can probably do this https://github.com/vwxyzjn/cleanrl/blob/b558b2b48326d8bb8da7fa853914576ae4610f53/cleanrl/ppo_continuous_action.py#L325

there are some ongoing gymnasium migration stuff that might have broken the Wandb gym integration.

**Comment by tbasaklar** [comment]
I've also realized that if I change the command from: 

`python cleanrl/ppo.py --seed 1 --env-id CartPole-v1 --track --capture-video`

to

`python cleanrl/dqn.py --seed 1 --env-id CartPole-v1 --track --capture-video`

It works! I wondered why that happened and realized it was because of the gymnasium migration, as you mentioned.
For dqn.py, the migration to the gymnasium is completed, I guess, and that's why it works.

I've also added the following condition (in ppo_continuous_action.py) to ppo.py. It works now!

Thanks,

**Comment by tbasaklar** [comment]
Hi, 

I appreciate the work, and I really love to work with cleanRL, especially for research purposes. When do you think you can complete the gymnasium migration for all algorithms? It is hard to follow the examples when they give errors related to Gymnasium and Gym discrepancies.

Best,

**Comment by vwxyzjn** [comment]
#424 should have completed the migration for almost all.
